### PR TITLE
Initial documentation on Rinkeby Nitro upgrade

### PR DESCRIPTION
**Pull request by joshuacolvin0**

### CODE REVIEW

File: docs/Running_Goerli_Nitro_Node.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-id: Running_Nitro_Node
```
Comment (fredlacs) [inline docs/Running_Goerli_Nitro_Node.md:2]: would be good for us to add a redirect from the old page to the new when this gets included to the docs page

Comment (joshuacolvin0) [reply docs/Running_Goerli_Nitro_Node.md]: Good point, added disambiguation page at same link

File: docs/Running_Rinkeby_Nitro_Node.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+id: Running_Rinkeby_Nitro_Node
+title: Running full Nitro node for Arbitrum Rinkeby TestNet after Nitro upgrade
+sidebar_label: Running a Rinkeby Nitro Node
+---
+
+Note: Rinkeby Nitro will not work until after the Nitro upgrade scheduled for Thursday, July 28th.
```
Comment (PlasmaPower) [inline docs/Running_Rinkeby_Nitro_Node.md:7]: Do you mean Arbitrum Classic nodes on Rinkeby will stop working?

**Review by PlasmaPower — APPROVED** [review]
LGTM

File: docs/Running_Rinkeby_Nitro_Node.md (hunk 2 above)
Comment (joshuacolvin0) [reply docs/Running_Rinkeby_Nitro_Node.md]: updated to be clearer

**Pull request merged**